code/dashboard-2/__tests__/utils/index.test.ts

```diff
@@ -1,28 +1,214 @@
-import { isValidateClusterName } from '../../src/util'
+import {
+  breakText,
+  isValidateClusterName,
```

Comment:
"isValidate"?  I would have expected "isValid" or "validate", unless the combination is some sort of convention I've not seen before.